docs/source/reporters.rst

```diff
@@ -69,6 +69,7 @@ At the moment, the following reporters are built-in:
 - **prowl**: Send a detailed notification via prowlapp.com
 - **pushbullet**: Send summary via pushbullet.com
 - **pushover**: Send summary via pushover.net
+- **sendmail**: Pipe a message to the system sendmail command
```

Comment:
There isn't a reporter called `sendmail`, its a method of the `email` reporter.

I think we need a new section like [this one](https://urlwatch.readthedocs.io/en/latest/reporters.html#e-mail-via-gmail-smtp), but titled "E-Mail via sendmail" or something.

Reply:
@Jamstah done!